radish/documentors/__main__.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides a script to document Radish step implementations
+"""
+
+from sys import exit
+
+from os.path import split
+
+from docopt import docopt
+
+from radish import __VERSION__
+
+from radish.documentors.sphinx.multipage import MultiPageSphinxDocumentSet
+from radish.documentors.sphinx.singlepage import SinglePageSphinxDocument
+from radish.documentors.steps_writer import StepsDocumentWriter
+
+from radish.errororacle import error_oracle, enable_excepthook
+
```

Comment:
I'm kind of picky for some formatting stuff (I know I've done horrible things in this code base - however, I'm here to improve the situation) ... I like the separation between the `radish.documentors` imports with a blank line from the others. I'd like to keep the others grouped though:

```python
from radish.errororacle import error_oracle, enable_exepthook
from radish.exceptions import RadishError
from radish.loader import load_module_assets
```